CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Tonomy-ID Contributor's Guide
+A big welcome and thank you for considering contributing to Tonomy! It is people like you that help shape the future where your identity is in your hands.
+
+Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved. It also communicates that you respect the time of the developers managing and developing these open source projects. In return, we will reciprocate this by adressing your issue, assessing changes and helping you finalize your pull requests.
+
+## Quicklinks
+* [Getting Started](#getting-started)
+* [Tonomy ID Workshop](#tonomy-id-workshop)
+* [Design](#design)
+* [General Practices](#general-practices)
```

Comment:
lets add a section here with an overview of our repositories and how they work together. its currently in the README.md L24 + L42-50